src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const object of actions) {
+    switch (object.type) {
+      case 'addProperties':
+        Object.assign(state, object.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const keyRemove of object.keysToRemove) {
+          delete state[keyRemove];
+        }
+        break;
+
+      case 'clear':
+        for (const keyClear in state) {
+          delete state[keyClear];
+        }
+        break;
+    }
+  }
+
+  return state;
```

Comment:
The function should change the `state` basing on the given `actions` array. there isn't necessary to return something.

Reply:
yes